cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -80,6 +80,12 @@ the cloud specific control loops shipped with Kubernetes.`,
 			}
 
 		},
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				return fmt.Errorf("%q does not take any arguments, got %q", cmd.CommandPath(), args[0])
```

Comment:
would it be clearer to include all the unexpected arguments, like this:

```suggestion
				return fmt.Errorf("%q does not take any arguments, got %q", cmd.CommandPath(), args)
```

Reply:
updated.